org.cispa.Scyther.json

```diff
@@ -0,0 +1,139 @@
+         "modules":[
+            {
+               "name":"wxwidgets",
+               "rm-configure":true,
+               "config-opts":[
+                  "--enable-shared",
+                  "--disable-static",
+                  "--enable-unicode",
+                  "--enable-display",
+                  "--enable-propgrid",
+                  "--enable-stc",
+                  "--with-gtk=2",
+                  "--with-opengl",
+                  "--with-libjpeg",
+                  "--with-libpng",
+                  "--with-zlib",
+                  "--disable-webkit",
+                  "--disable-webview",
+                  "--disable-ribbon"
+               ],
+               "sources":[
+                  {
+                     "type":"archive",
+                     "url":"https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.2/wxWidgets-3.0.2.tar.bz2",
+                     "sha256":"346879dc554f3ab8d6da2704f651ecb504a22e9d31c17ef5449b129ed711585d"
+                  },
+                  {
+                     "type":"shell",
+                     "commands":[
+                        "cp -p /usr/share/automake-*/config.{sub,guess} .;",
+                        "autoconf -f -B \"build/autoconf_prepend-include\";"
+                     ]
+                  }
+               ],
+               "post-install":[
+                  "install -p -D -m 0644  \"docs/gpl.txt\" \"docs/lgpl.txt\" \"docs/licence.txt\" -t \"${FLATPAK_DEST}/share/licenses/wxwidgets/\";"
+               ]
+            }
+         ]
+      },
+      {
+         "name":"scyther",
+         "buildsystem":"simple",
+         "build-commands":[
+            "export CC=gcc && make",
+            "mkdir -p /app/bin/scyther",
+            "cp -r ./* /app/bin/scyther"
+         ],
+         "sources":[
+            {
+               "type":"git",
+               "url":"https://github.com/cascremers/scyther",
+               "commit":"bf4bd3e07bd0c10a39c3e2324b0eb416d504f6fc"
+            },
+            {
+               "type":"patch",
+               "path":"./BuildUnix.patch"
+            }
+         ]
+      },
+      {
+         "name":"runner",
+         "buildsystem":"simple",
+         "build-commands":[
+            "install -D runner.sh /app/bin/runner.sh",
+            "install -Dm644 org.cispa.Scyther.appdata.xml /app/share/appdata/org.cispa.Scyther.appdata.xml",
+            "install -Dm644 org.cispa.Scyther.svg /app/share/icons/hicolor/scalable/apps/org.cispa.Scyther.svg",
+            "install -Dm644 org.cispa.Scyther-symbolic.svg /app/share/icons/hicolor/symbolic/apps/org.cispa.Scyther-symbolic.svg",
+            "install -Dm644 org.cispa.Scyther.desktop /app/share/applications/org.cispa.Scyther.desktop"
+         ],
+         "sources":[
+            {
+               "type":"file",
+               "path":"runner.sh"
+            },
+            {
+               "type":"file",
+               "path":"org.cispa.Scyther.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html